docs/guides/security/authentication/linode-manager-security-controls/index.md

```diff
@@ -51,9 +51,9 @@ Authy stores your authentication tokens (hashed for security) on their servers.
 Enable two-factor authentication to start using it with your Linode account.
 
 1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com).
-1.  Select the **My Profile** link by clicking on your username at the top of the page:
+1.  Select the **My Settings** link by clicking on your username at the top of the page:
 
-   ![Select 'My Profile'](security-my-profile.png "Select 'My Profile'")
```

Comment:
Please resize the image and indent it because the steps are renumbered:https://deploy-preview-3873--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/linode-manager-security-controls/